.pre-commit-config.yaml

```
@@ -70,3 +70,8 @@ repos:
         files: ^js/.*$
       - id: prettier
         types: [yaml]
+        
+  - repo: https://github.com/koalaman/shellcheck-precommit
+    rev: v0.7.2
```

Please use the latest version `v0.8.0` to maintain consistency among all the repos.
```suggestion
    rev: v0.8.0
```